client/js/lounge.js

```diff
@@ -133,6 +133,14 @@ $(function() {
 					data: target.data("id"),
 				});
 			}
+			if (target.hasClass("channel") && utils.hasRoleInChannel($(`#chan-${target.data("id")}`))) {
+				output += templates.contextmenu_item({
+					class: "list",
+					action: "banlist",
+					text: "Show ban list",
```

Comment:
Could you do "List banned users"? That is a bit more explanatory to me and fits better with the "List all channels" wording we did for #1802.

Reply:
Makes sense, done